app/src/main/java/edu/uwstout/p2pchat/FileTransfer/SendDataService.java

```diff
@@ -32,8 +32,12 @@
  *
  * @author VanderHoevenEvan (Evan Vander Hoeven)
  */
-public final class SendDataService extends IntentService
+public class SendDataService extends IntentService
 {
+    static boolean isTesting = false;
```

Comment:
classes should never be aware that they are in a testing environment. This becomes a potential example of the observer effect, where testing it changes its functionality, which means you're no longer testing the send data service as it would be in production.